universal-application-tool-0.0.1/test/support/TestQuestionBank.java

```diff
@@ -343,6 +349,17 @@ private Question applicantSeason(QuestionEnum ignore) {
     return maybeSave(definition);
   }
 
+  // Static
+  private Question applicantStatic(QuestionEnum ignore) {
```

Comment:
nit: remove `applicant` from this name. Maybe just call it `staticContent` or something? the `applicant` prefix in the other questions here are describing the questions like "applicant's name" (vs. household member name).